app/src/main/java/org/sagebionetworks/research/mpower/room/RoomScheduledActivity.kt

```diff
@@ -0,0 +1,130 @@
+package org.sagebionetworks.research.mpower.room
+
+import android.arch.persistence.room.ColumnInfo
+import android.arch.persistence.room.Embedded
+import android.arch.persistence.room.Entity
+import android.arch.persistence.room.Index
+import android.arch.persistence.room.PrimaryKey
+import com.google.gson.annotations.SerializedName
+import org.joda.time.DateTime
+import org.sagebionetworks.bridge.rest.model.ActivityType
+import org.sagebionetworks.bridge.rest.model.ScheduleStatus
+
+import java.util.ArrayList
+
+@Entity
+data class RoomScheduledActivity(@SerializedName("guid") @PrimaryKey var guid: String) {
+    @SerializedName("schedulePlanGuid")
```

Comment:
Do you mean I should name this "EntityScheduledActivity" instead of "RoomScheduledActivity".  Or do you mean I shouldn't include the @SerializedName? I did the SerializedName stuff only so I could easily go to/from ScheduledActivity bridge model by simply using GSON